Code/Editor/EditorViewportSettings.cpp

```diff
@@ -61,6 +61,7 @@ namespace SandboxEditor
     constexpr AZStd::string_view CameraDefaultStartingYaw = "/Amazon/Preferences/Editor/Camera/DefaultStartingYaw";
     constexpr AZStd::string_view CameraNearPlaneDistanceSetting = "/Amazon/Preferences/Editor/Camera/NearPlaneDistance";
     constexpr AZStd::string_view CameraFarPlaneDistanceSetting = "/Amazon/Preferences/Editor/Camera/FarPlaneDistance";
+    constexpr AZStd::string_view CameraFovSetting = "/Amazon/Preferences/Editor/Camera/DefaultFOV";
```

Comment:
update: It would be good to callout degrees in the name of this variable too `DefaultFovDegrees` ideally

Reply:
Also we probably shouldn't call this 'Default' as the value will change, this should just be `FovDegrees`